WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -16,12 +16,12 @@ def competition_message_for_user(competition, user)
   end
 
   def announced_class(days_announced)
-    level = [Competition::ANNOUNCED_DAYS_WARNING, Competition::ANNOUNCED_DAYS_DANGER].select {|d| days_announced > d}.count
+    level = [Competition::ANNOUNCED_DAYS_WARNING, Competition::ANNOUNCED_DAYS_DANGER].select { |d| days_announced > d }.count
```

Comment:
This one I actually would prefer to leave on. Basically because I don't want to be "that guy" writing spacing comments on people's PRs.